content/02-Platform/Frontend/00-Overview.md

```diff
@@ -0,0 +1,25 @@
+# Front end architecture
+
+Welcome to the VA modern platform front end overview. The goal of this article is to provide a high level overview of our front end architecture to get your oriented in the [vets-website](https://github.com/department-of-veterans-affairs/vets-website) repository.
+
+The overall architecture for vets.gov is that we have an API backend written in Ruby on Rails ([vets-api](https://github.com/department-of-veterans-affairs/vets-api)) and a static front end, written in HTML, CSS, and Javascript (`vets-website`). There's no server side rendering or processing for the front end; the build process compiles everything into static resources which are served to users.
+
+## Build process
+
+The build process for this is controlled by a tool called [Metalsmith](http://www.metalsmith.io/), which is a pluggable Node.js static site generator. In general, everything in m Metalsmith is a plugin, so if you're looking for specific parts of the build process, expect to check out a plugin or two. We use Metalsmith because it's flexible and because it's Javascript based, it's easy to integrate with the rest of our tools.
+
+The build process has two primary kinds of functionality that it processes: static content pages and Javascript applications. Most of Vets.gov consists of content that lives in Markdown files and is converted into html by a Metalsmith plugin. There are shared templates and other enhancements to make this content easier to structure. This content lives in `content/pages`, which also provides the basic url structure of the site.
+
+## Javascript applications
+
+Javascript applications are the other type of functionality in vets-website. These applications are written in [React](https://reactjs.org/) and [Redux](https://redux.js.org/) and live in `src/js/` in `vets-website`. For each app you'll find a `manifest.json` file that contains application information. You'll find the root url of the application, which you can use to navigate to that application on the site. These applications also usually have client side routes, so opening that root url may allow you to click to other pages that are fully client side, and not based on pages in `content/pages`.

```

Comment:
Would it make sense to also list `reactjson-schema-form` ~here~ in the next paragraph?